doc/sphinx/proof-engine/tactics.rst

```diff
@@ -901,6 +893,18 @@ quantification or an implication.
    This clears the hypothesis :n:`@ident` and all the hypotheses that depend on
    it.
 
+.. tacv:: clearbody @ident
+   :name: clearbody
+
+   This tactic expects :n:`@ident` to be a local definition then clears its
```

Comment:
```
:token:`ident`
```
should generally be preferred over
```
:n:`@ident`
```
(cf. https://github.com/coq/coq/tree/master/doc/sphinx#coq-roles)
and I'm not sure that the word "then" is appropriate in this case (there is no "if"). I know that you didn't write this but since you are moving these lines, why not improve them at the same time?